dao/src/test/java/com/fitgoal/dao/impl/utils/DocumentUtils.java

```diff
@@ -0,0 +1,58 @@
+package com.fitgoal.dao.impl.utils;
+
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
+import org.bson.Document;
+import org.testcontainers.shaded.org.apache.commons.io.IOUtils;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.fitgoal.dao.util.AuditDtoFields.DATE;
+
+@UtilityClass
+class DocumentUtils {
+
+    private static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+
+    List<Document> parseDocuments(@NonNull String resourceName) {
+        try {
+            String docJson = IOUtils.resourceToString(resourceName, StandardCharsets.UTF_8);
+
+            Object content = Document.parse("{\"json\":" + docJson + "}").get("json");
+            List<Document> documents = (List<Document>) content;
+            return documents
+                    .stream()
+                    .map(document -> fixDate(
+                            document, convertDate(document.getString(DATE))))
+                    .collect(Collectors.toList());
+        } catch (IOException ex) {
+            throw new RuntimeException("Can't handle resource: " + resourceName, ex);
+        }
+    }
+
+    private LocalDateTime convertDate(String date) {
+        LocalDateTime localDateTime = null;
+        try {
+            localDateTime = SIMPLE_DATE_FORMAT
+                    .parse(date)
+                    .toInstant()
+                    .atZone(ZoneOffset.UTC)
+                    .toLocalDateTime();
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        return localDateTime;
+    }
+
+    private Document fixDate(Document document, LocalDateTime date) {
```

Comment:
if you accept my refactoring proposal, remove this method

Reply:
removed